lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/ActiveContextBridge.java

```diff
@@ -27,21 +27,19 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-public class ActiveContextBridge extends NativeBridge implements Identifiable {
+public final class ActiveContextBridge extends NativeBridge implements Identifiable {
   private static final Logger LOG = Logger.getLogger(ActiveContextBridge.class.getName());
 
   private final ActiveContext jactiveContext;
   private final AvroConfigurationSerializer serializer;
-  private final String contextId;
```

Comment:
Both cpntextId and evaluatorId are used in cpp bridge code. It references the fields directly rather than GetId method. So those two fields need to stay.